### PR TITLE
feat(admin-server): add sentry reporting

### LINKED ISSUE #4242: Sentry reporting for admin server
Add Sentry config option and reporting of errors to admin server.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1186)
┆Issue Number: FXA-1186

### PR DESCRIPTION
**Pull request by bbangert**
Because:

* We have no sentry reporting of internal errors.

This commit:

* Captures internal resolver errors to Sentry.

Closes #4242

### CODE REVIEW

File: packages/fxa-admin-server/src/test/lib/sentry.spec.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      undefined,
+      undefined,
+      ['resolver', 'field'],
+      originalError
+    );
+    reportGraphQLError(false, logger, err);
+    const errResult = mockCaptureException.args[0][0];
+    assert.equal(errResult.message, 'boom');
+    const logSpy = (logger.error as unknown) as SinonSpy;
+    assert.isTrue(logSpy.calledOnceWith('graphql', { path: 'resolver.field', error: 'boom' }));
```
Comment (chenba) [inline packages/fxa-admin-server/src/test/lib/sentry.spec.ts]: Could additionally assert that mockScope.setContext have been called with the expected arguments.

File: packages/fxa-admin-server/src/lib/version.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    readJson(path.resolve(__dirname, '..', '..', '..', 'package.json'));
+
+  return {
+    commit,
+    source,
+    version: packageInfo?.version
+  };
+}
+
+export const version = getVersionInfo();
```
Comment (chenba) [inline packages/fxa-admin-server/src/lib/version.ts]: Is this file from somewhere?  I don't see any tests for, but if it's working code from somewhere 🤷‍♂

Comment (bbangert) [reply packages/fxa-admin-server/src/lib/version.ts]: It is, its from event-broker which is from auth-server. Ideally it should be in fxa-shared.

File: packages/fxa-admin-server/src/test/lib/sentry.spec.ts (hunk 1 above)
Comment (bbangert) [reply packages/fxa-admin-server/src/test/lib/sentry.spec.ts]: Added that.

**Pull request merged**